### PR TITLE
20230609 guide processing rebuild

### PR DESCRIPTION
**Pull request by wkande**
This is a rebuild of #498 which had conflicts I could not resolve.

### CODE REVIEW

**Comment by wkande** [comment]
I any of you get a chance, please look at this guide in production as I am merging it. I need it online today. It is in pretty good shape now and I do not believe there are errors but your opinions would help. I anyone wants to create a new PR to makes changes please do.

File: docs/guides/airnode/post-processing/index.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+In this guide you will use `postProcessingSpecifications` to get the average
+price of the API3 token and its average market cap for the last seven days. This
+will be calculated after calling the CoinGecko operation
+`/coins/{coin}/market_chart`.
+
+## 1. Setup a Docker container to run Airnode
+
+A local Docker container running Airnode is needed. Go to the guide
+[Deploying Airnode locally using Docker](/guides/airnode/deploy-airnode/deploy-container/index.md)
+to set one up. Once the container is running, return to step #2 below.
```
Comment (dcroote) [inline docs/guides/airnode/post-processing/index.md:38]: I think it's a bit strange to have them deploy an Airnode only to take it down in Step 4. Why not just say run steps 1-3 of that tutorial?

File: docs/guides/airnode/post-processing/index.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+```
+
+## 4. Remove the existing Docker container
+
+The Docker container you created in step #1 must be deleted and a new one
+created which uses the newer `config.jon` file. From within the Docker desktop
+app find the docker container named **quick-start-container-airnode** and delete
+it.
+
+<img src="./src/delete-container.png"/>
```
Comment (dcroote) [inline docs/guides/airnode/post-processing/index.md:117]: I suggest deleting this step per my comment above.

File: docs/guides/airnode/post-processing/index.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+  --volume "$(pwd):/app/config" \
+  --name quick-start-container-airnode \
+  --publish 3000:3000 \
+  api3/airnode-client:latest
+WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
+[2023-06-06 21:49:44.565] INFO Gateway "httpSignedDataGateway" not enabled.
+[2023-06-06 21:49:44.586] INFO Gateway "oevGateway" not enabled.
+[2023-06-06 21:49:44.596] INFO HTTP (testing) gateway listening for request on "http://localhost:3000/http-data/01234567-abcd-abcd-abcd-012345678abc/:endpointId"
+[2023-06-06 21:49:44.607] INFO API gateway server running on "http://localhost:3000"
+
```
Comment (dcroote) [inline docs/guides/airnode/post-processing/index.md:170]: I would suggest limiting this code block to only the relevant line, which is the one with the gateway URL, and changing the text to highlight that the user needs this URL for the next step.

File: docs/guides/airnode/post-processing/index.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+determine what data should go into the `values` field. Airnode then encoded the
+values into the `encodedValue` field suitable for use on-chain.
+
+## 7. Verify `encodedValue`
+
+Verify that `encodedValue` actually contains the two numbers in the `values`
+field. Copy and paste the `encodedValue` string you received after executing the
+Airnode endpoint into the form below and select the **Decode** button.
+
+<EthersAbiCoder types="int256,int256"/>
```
Comment (dcroote) [inline docs/guides/airnode/post-processing/index.md:245]: This is cool 👍

File: docs/guides/airnode/post-processing/index.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+
+## Summary
+
+In this guide you extended the data from the API operation with additional
+summary values. For more examples on how to use `preProcessingSpecifications`
```
Comment (dcroote) [inline docs/guides/airnode/post-processing/index.md:250]: Suggestion: "summary values using post processing. For more..."

**Review by dcroote — COMMENTED** [review]
Nice guide. Some suggestions per your request.

File: docs/guides/airnode/post-processing/index.md (hunk 1 above)
Comment (wkande) [reply docs/guides/airnode/post-processing/index.md:38]: Not sure I see how that will work. The reader needs to create the secrets.env file which its completed in The quick start tutorial. By running that tutorial to completion we know that they have all files set up and the container works. Then the new config.json file gets added into the project folder and a replacement Docker container get created, the new config uses a new API provider operation. I suppose we could tell them to run the quick start tutorial using the new config to begin with.

File: docs/guides/airnode/post-processing/index.md (hunk 2 above)
Comment (wkande) [reply docs/guides/airnode/post-processing/index.md:117]: We can do this if the quick start tutorial uses the new configuration file.

File: docs/guides/airnode/post-processing/index.md (hunk 5 above)
Comment (wkande) [reply docs/guides/airnode/post-processing/index.md:250]: OK

File: docs/guides/airnode/post-processing/index.md (hunk 1 above)
Comment (dcroote) [reply docs/guides/airnode/post-processing/index.md:38]: But why not say run steps 1-3 of the quick start tutorial, but skip the `config.json` setup? This is clear, whereas deploying a container as part of another tutorial only to stop the container, replace a file, and deploy again is a lot of extra steps.

**Pull request merged**